src/transformers/LogicalAssignmentTransformer.ts

```diff
@@ -0,0 +1,164 @@
+import type {HelperManager} from "../HelperManager";
+import type {Token} from "../parser/tokenizer";
+import {TokenType as tt} from "../parser/tokenizer/types";
+import type TokenProcessor from "../TokenProcessor";
+import type RootTransformer from "./RootTransformer";
+import Transformer from "./Transformer";
+
+const LOGICAL_OPERATORS = ["&&=", "||=", "??="];
+
+interface ComputedAccess {
+  // A snapshot of the result just before the `[` of the computed access
+  snapshot: {
+    resultCode: string;
+    tokenIndex: number;
+  };
+  // The result code position at the start of the computed property access
+  start: number;
+}
+
+export default class LogicalAssignmentTransformer extends Transformer {
+  // This stack stores the state needed to transform computed property access
+  private readonly computedAccessStack: Array<ComputedAccess> = [];
```

Comment:
Switched to a manual stack instead of relying on the JS execution stack. This seemed to avoid some of the competition going on with other processors, but could switch back if you think the old approach is better.